### PR TITLE
Feature/agents online

### PR DESCRIPTION
**Pull request by gabrielperales**

### CODE REVIEW

File: ios/RNZendeskChatModule.m [hunk 1]
```diff
@@ -201,4 +201,25 @@ - (void) dismissChatUI {
 	});
 }
 
+RCT_EXPORT_METHOD(areAgentsOnline:
+	(RCTPromiseResolveBlock) resolve
+	rejecter: (RCTPromiseRejectBlock) reject) {
+
+  [ZDKChat.accountProvider getAccout:^(ZDKChatAccount *account, NSError *error) {
```
Comment (fbartho) [inline ios/RNZendeskChatModule.m:208]: Is there a typo here?

Comment (gabrielperales) [reply ios/RNZendeskChatModule.m:208]: yes, looks like it is

**Comment by gabrielperales** [comment]
@fbartho Tested on iOS and it works :)

**Review by fbartho — APPROVED** [review]
This code looks pretty good, and I'll try to integrate it!

File: ios/RNZendeskChatModule.m [hunk 2]
```diff
@@ -201,4 +201,23 @@ - (void) dismissChatUI {
 	});
 }
 
+RCT_EXPORT_METHOD(areAgentsOnline:
+	(RCTPromiseResolveBlock) resolve
+	rejecter: (RCTPromiseRejectBlock) reject) {
+
+  [ZDKChat.accountProvider getAccount:^(ZDKChatAccount *account, NSError *error) {
+		if (account) {
```
Comment (fbartho) [inline ios/RNZendeskChatModule.m:209]: If there is no account, this promise never resolves.

I'm going to add a code-snippet to reject in that situation.

File: android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java [hunk 3]
```diff
@@ -377,4 +378,29 @@ public void update(ChatState chatState) {
             }
         });
     }
+
+	@ReactMethod
+	public void areAgentsOnline(Promise promise) {
```
Comment (moorjaniNitesh95) [inline android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java:383]: No import for Promise, Resulting in Crash, Please fix it, you cant use Promise

Comment (fbartho) [reply android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java:383]: This is fixed in v0.4.1-beta.7 and v0.4.1-beta.8. What version are you using?

Comment (fbartho) [reply android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java:383]: Oh! Maybe it’s not. I’ll take a look tomorrow. We had some build issues due to jitpack.io being offline yesterday.

Comment (moorjaniNitesh95) [reply android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java:383]: yes its not and after npm install 0.4.1-beta.8 is getting installed and causing build error, you need to fix this, in a new PR

Comment (fbartho) [reply android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java:383]: Active fixes for 0.4.1 can be found in #85 -- `0.4.1-beta.9` is published and now building in our CI. If further fixes are needed, I'll publish more betas until it seems ready.

Comment (fbartho) [reply android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java:383]: Beta.9 passed compilation, so I'm shipping 0.4.1! -- If you find bugs, please report them.

Comment (moorjaniNitesh95) [reply android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java:383]: Nice work @fbartho !!

**Pull request merged**